modules/prebidServerBidAdapter/index.js

```diff
@@ -475,7 +475,9 @@ export function PrebidServer() {
           if (isValid) {
             bidRequests.forEach(bidderRequest => events.emit(EVENTS.BIDDER_DONE, bidderRequest));
           }
-          if (shouldEmitNonbids(s2sBidRequest.s2sConfig, response)) {
+          const { emitSeatNonBid, emitAtag } = shouldEmitAnalyticsEvent(s2sBidRequest.s2sConfig, response)
+          // should be deprecated in Prebid 10
```

Comment:
can you elaborate?

Reply:
We would like to deprecate the seat_non_bid event in favor of the pbs_analytics event. That way adapters only need to watch for the one event.